pre2p/genesis.go

```diff
@@ -0,0 +1,70 @@
+package pre2p
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"pocket/pre2p/types"
+	"time"
+)
+
+// TODO(olshansky): This is an interim genesis structure that will be replaced with a real one. It is the bare minimum for prototyping purposes.
+type P2PGenesis struct {
+	GenesisTime time.Time          `json:"genesis_time"`
+	AppHash     string             `json:"app_hash"`
```

Comment:
No idea why P2PGenesis needs an appHash... Can this be removed?

Reply:
The hash of the last block for the V0 blockchain when we do the migration?

Reply:
Yes but why for P2P? Consensus might need one.